### DISCUSSION TITLE
Passing response of one api to another in artillery api tests.

### DISCUSSION THREAD

**Discussion by BhavnaSoni1991** [artilleryio/artillery#2775] [Artillery]
Hello Team I am writing api performance tests using artillery and for some api the request payload has fields for which value is set from response of other api. Now since I am writing separate yml file for each endpoint is there a way to set ṭhe response fields of one api in any global variable which can be accessed in yml file of other endpoint?

**Comment by hassy** [comment]
Hi @BhavnaSoni1991 👋 If your API calls are in two separate files, the only way to pass values between the executions of those calls is through some sort of external storage (disk, Redis, S3 or something like that). Is there a reason you're not chaining those requests in one flow? Then you could use `capture` actions to pass values between calls very easily.
